sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -385,6 +396,8 @@ private void setArchitectures(final @NotNull Device device) {
     if (device.getLanguage() == null) {
       device.setLanguage(Locale.getDefault().toString()); // eg en_US
     }
+
+    // likely to be applied only if applyScopeData
```

Comment:
the chances that the user opens the app again right away is high, so this won't change much